test/dotc/tests.scala

```diff
@@ -1,30 +1,29 @@
 package dotc
 
 import test._
-import dotty.partest._
-import org.junit.Test
-import org.junit.experimental.categories._
+import org.junit.{Before, Test}
 
+import scala.reflect.io.Directory
 import scala.io.Source
 
 // tests that match regex '(pos|dotc|run|java|compileStdLib)\.*' would be executed as benchmarks.
 class tests extends CompilerTest {
 
   def isRunByJenkins: Boolean = sys.props.isDefinedAt("dotty.jenkins.build")
 
+  val defaultOutputDir = "./out/"
+
   val noCheckOptions = List(
 //        "-verbose",
 //         "-Ylog:frontend",
 //        "-Xprompt",
 //        "-explaintypes",
 //        "-Yshow-suppressed-errors",
+        "-d", defaultOutputDir,
         "-pagewidth", "160")
```

Comment:
Note that I moved the `-d` option to `noCheckOptions`, so that the `dotc_core_nocheck` test case also generates class files under `./out/`, instead of the current directory.